src/main/resources/static/style.css

```diff
@@ -52,18 +55,21 @@ h3{
 	font-size: 40px;	
 }
 
-
-
 .container {
   width: 80%;
+  height: 200px;
```

Comment:
Avoid using pixels, where possible, to promote fluid layouts.  Unless you have a specific css for each screen density (wear, phone, tablet, screen)